GoMain/src/main/main.go

```diff
@@ -63,6 +65,7 @@ const (
 
 	cipherKeyFilePath = edgeDir + "/user/orchestration_userID.txt"
 	deviceIDFilePath  = edgeDir + "/device/orchestration_deviceID.txt"
+	mnedcServerConfig = edgeDir + "/client.config"
```

Comment:
How about make a folder for mnedc like below?
```suggestion
	mnedcServerConfig = edgeDir + "/mnedc/client.config"
```

Reply:
Done